solutions/standard/DA-types.md

```diff
@@ -0,0 +1,165 @@
+- `io_enabled`: Indicates whether IO-based autoscaling is enabled (default: `false`).
+- `io_over_period`: How long I/O usage is evaluated for autoscaling (default: `"15m"` (15 minutes)).
+- `rate_increase_percent`: The percentage increase in disk capacity when autoscaling is triggered (default: `10`).
+- `rate_limit_mb_per_member`: The limit in megabytes for the rate of disk increase per member (default: `3670016`).
+- `rate_period_seconds`: How long (in seconds) the rate limit is applied for disk (default: `900` (15 minutes)).
+- `rate_units`: The units to use for the rate increase (default: `"mb"` (megabytes)).
+
+
+### Memory options for auto_scaling
+
+The memory object within auto_scaling contains the following options. All options are optional.
+
+- `io_above_percent`: The percentage of I/O memory usage that triggers autoscaling (default: `90`).
+- `io_enabled`: Whether IO-based autoscaling for memory is enabled (default: `false`).
+- `io_over_period`: How long I/O usage is evaluated for memory autoscaling (default: `"15m"` (15 minutes)).
+- `rate_increase_percent`: The percentage increase in memory capacity that triggers autoscaling (default: `10`).
+- `rate_limit_mb_per_member`: The limit in megabytes for the rate of memory increase per member (default: `114688`).
+- `rate_period_seconds`: How long (in seconds) the rate limit is applied for memory (default: `900` (15 minutes)).
+- `rate_units`: The memory size units to use for the rate increase (default: `"mb"` (megabytes)).
+
+### Example autoscaling
+
+The following example shows values for both disk and memory for the `auto_scaling` input.
+
+```hcl
+{
+  "disk": {
+      "capacity_enabled": true,
+      "free_space_less_than_percent": 15,
+      "io_above_percent": 85,
+      "io_enabled": true,
+      "io_over_period": "15m",
+      "rate_increase_percent": 15,
+      "rate_limit_mb_per_member": 3670016,
+      "rate_period_seconds": 900,
+      "rate_units": "mb"
+  },
+  "memory": {
+      "io_above_percent": 90,
+      "io_enabled": true,
+      "io_over_period": "15m",
+      "rate_increase_percent": 10,
+      "rate_limit_mb_per_member": 114688,
+      "rate_period_seconds": 900,
+      "rate_units": "mb"
+  }
+}
+```
+
+## Configuration  <a name="configuration"></a>
+
+The Configuration variable tunes the Redis database to suit different use case. For more information, see [Configuration](https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-changing-configuration&interface=cli).
+
+- Variable name: `configuration`
+- Type: An object with `maxmemory`, `maxmemory-policy`, `appendonly`, `maxmemory-samples` and `stop-writes-on-bgsave-error` attributes
+
+### Options for auto_scaling
+
+The configuration object in the input contains the following options. All options are optional.
+
+- `maxmemory`: Determines the amount of data that you can store in Redis (default: `80`).
```

Comment:
Can we do better on this? https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-redis-cache

Determines the amount of data that you can store in Redis as a percentage of the deployments memory.

Combines the information from notes and default at https://cloud.ibm.com/docs/databases-for-redis?topic=databases-for-redis-changing-configuration.

Reply:
addressed